shunting-yard-common/src/main/java/com/hotels/shunting/yard/common/emitter/AbstractMetaStoreEventListener.java

```diff
@@ -45,9 +45,21 @@
 import com.hotels.shunting.yard.common.messaging.Message;
 import com.hotels.shunting.yard.common.messaging.MessageTask;
 import com.hotels.shunting.yard.common.messaging.MessageTaskFactory;
+import com.hotels.shunting.yard.common.metrics.MetricsConstant;
+import com.hotels.shunting.yard.common.metrics.MetricsHelper;
 
 public abstract class AbstractMetaStoreEventListener extends MetaStoreEventListener {
-  private static final Logger LOG = LoggerFactory.getLogger(AbstractMetaStoreEventListener.class);
+  private static final Logger log = LoggerFactory.getLogger(AbstractMetaStoreEventListener.class);
+
+  static void success() {
```

Comment:
What's the reason these two methods are static?

Reply:
they don't depend in the class state